nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -398,10 +398,14 @@ def add_electrodes(
     for name in elec_columns_append:
         _ = elec_columns.pop(name)
 
+    # add channel_id column
+    nwbfile.add_electrode_column(name="channel_id", description="channel_id", index=False)
+
     for j, channel_id in enumerate(checked_recording.get_channel_ids()):
         if channel_id not in nwb_elec_ids:
```

Comment:
Note that `nwb_elec_ids=nwbfile.electrodes.id.data[:]`.

That is, maybe we should have?
```suggestion
        if checked_recording.id_to_index(channel_id) not in nwb_elec_ids:
```